src/components/common/PaginatedTxns/index.tsx

```diff
@@ -49,7 +49,7 @@ const TxPage = ({
         </Box>
       )}
 
-      {page && page.results.length > 0 && <TxList items={page.results} />}
+      {!!page?.results.length && <TxList items={page.results} />}
```

Comment:
i fixed it on my branch before it got fixed on main and i think it looks tidier with optional chaining, lmk if I should revert the fix

Reply:
Chaining for sure, but we should avoid `!!` as it's somewhat cryptic.
Please use `page?.results.length > 0`

Reply:
It seems like `page?.results.length > 0` check is not enough for typescript to ensure that `page` is defined, same for `Boolean(page?.results.length)`. I'll just revert the change then